ui/packages/catalog-ui-search/src/main/webapp/js/filter.js

```diff
@@ -64,26 +64,19 @@ function checkToken(token, filter) {
 }
 
 function matchesILIKE(value, filter) {
-  const valueToCheckFor = filter.value.toLowerCase()
+  filter.value = filter.value.toLowerCase()
```

Comment:
maybe an naive question, but I am going to ask anyway.... by modifying the filter.value, do we end up changing the user's input they made?  I am hoping the answer is no, but I have seen it leak before (like in default search titles)

Reply:
Double-checked this and the value is not leaking back to the UI or changing the user's entered input